functions/Get-DBAQueryPlan.ps1

```diff
@@ -0,0 +1,93 @@
+﻿function Get-DbaQueryPlan {
+<#
+    .SYNOPSIS
+        Provides information about adhoc and prepared plan cache usage
+
+    .DESCRIPTION
+        "Checks ahoc and prepared plan cache for each database, if over 100 MBS should 
+        consider you using Remove-DbaQueryPlan to clear the plan caches or turning on
+        optimize for adhoc workloads configuration is running 2008 or later."
+
+        References: https://www.sqlskills.com/blogs/kimberly/plan-cache-adhoc-workloads-and-clearing-the-single-use-plan-cache-bloat/
+
+        Note: This command returns results from all SQL instances on the destionation server but the process
+        column is specific to -SqlInstance passed.
+
+    .PARAMETER SqlInstance
+        Allows you to specify a comma separated list of servers to query.
+
+    .PARAMETER SqlCredential
+        Allows you to login to the SQL instance using alternative credentials.
+
+    .PARAMETER Threshold
+        Memoery used threshold.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Memory
+        Author: Tracy Boggiano, databasesuperhero.com
+
+        dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+        Copyright (C) 2016 Chrissy LeMaire
+        License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+    .LINK
+        https://dbatools.io/Get-DbaQueryPlan
+
+    .EXAMPLE
+        Get-DbaQueryPlan -SqlInstance sql2017
+
+        Logs into the SQL Server instance "sql2017" and also the Computer itself (via WMI) to gather information
```

Comment:
Seems like these may be from template usage...please update according to the command. 

Something like this would be sufficient: `Returns plan cache usage information for SQL Server instance sql2017`